clients/fides-js/src/components/fides.css

```diff
@@ -1096,6 +1093,10 @@ div#fides-overlay-wrapper .fides-toggle .fides-toggle-display {
     .fides-banner-button-group {
     padding-left: 0;
   }
+  .fides-banner-secondary-actions {
```

Comment:
Did this class exist before? If so, are we certain it won't break current UI?

Reply:
the class existed but looked broken in mobile view with the "Manage Preferences" button appearing _below_ the language switcher. This update puts the language switcher at the bottom as expected. Happy to remove this change from this current PR and create a "bug" if we think that's a safer bet.

Reply:
this is fine to keep here, just wanted to make sure we keep any changes to a min. But this fixes an existing issue, so this is fine!